Automattic/wp-calypso
# people(followers): use `react-query` to fetch data & remove followers

This PR is branched upon #50262 to make it easier to review. It should be rebased on `trunk` as #50262 gets in.

#### Changes proposed in this Pull Request

* People: migrate email-/followers to use `react-query`

#### Testing instructions

1. Go to the people section of a site and choose _Followers_
2. Make sure the list of followers shows up
3. If you scroll down on a page with more than 100 followers you should see another request with incremented `page` param
4. If you switch sites, make sure request params are reset and the list loads fine
5. Try to remove followers, are they deleted from the list and don't appear again on a reload?

Follow the steps above for _Email Followers_ as well.
